crates/neon/src/types_impl/private.rs

```diff
@@ -31,13 +34,13 @@ pub(crate) unsafe fn prepare_call<'a, 'b, C: Context<'a>>(
 pub trait ValueInternal: TransparentNoCopyWrapper + 'static {
```

Comment:
`ValueInternal::from_local` could accept safe wrappers, but it would need to remain `unsafe` since it doesn't perform type checking.